src/modules/fundingManager/oracle/interfaces/IOraclePrice_v1.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+/**
+ * @title   Oracle Price Interface.
```

Comment:
Natspec section needs to be copied from the contract now that that one was changed. also no dot at the end of a title 🙏

Reply:
there isn't a `OraclePrice_v1` implementation, so what do you suggest, @marvinkruse?

This interface is extended in `interface ILM_ManualExternalPriceSetter_v1 is IOraclePrice_v1 `

Reply:
dot removed